test/run-drun/stable-region-blob.mo

```diff
@@ -0,0 +1,118 @@
+import P "mo:⛔";
+import Region "stable-region/Region";
+actor {
+
+  stable var n : Nat64 = 0;
+//  stable var r : Region = Region.new();
+  let next_id = Region.next_id();
+  assert next_id == 2;
```

Comment:
@crusso Even though it's trivial, it felt good to get this test to pass. :)

More substantively, I was able to get a bit more comfortable changing the special comments that go to `run.sh` and to `drun`.